monitoring/alertmanager/alertmanager-svc.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: alertmanager-tls
+  labels:
+    name: alertmanager
+  name: alertmanager
+  namespace: "redhat-ods-monitoring"
```

Comment:
redhat-ods-monitoring namespace? I don't see such a namespace on any of our clusters.

Reply:
redhat-* wont exists in any ODH cluster, and the monitoring/*/* will only be applied when detect it as a redhat managed cluster. 
if this makes upstream more confusing, we can apply this change directly in downstream "odh-manifests" repo instead.
the reason to have it here is only to make the sync of odh-manifests from upstream to downstream easier.

Reply:
Yeah, as long as this is an overlay here, it's cool. But I think you all should really get together with your SRE / site reliability engineer, with @VannTen and others to rethink your use of this prometheus here downstream.